packages/ui/src/common/ErrorModal.tsx

```diff
@@ -0,0 +1,159 @@
+/*
+ *
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ *  This program and the accompanying materials are made available under the terms of
+ *  the GNU Affero General Public License v3.0. You should have received a copy of the
+ *  GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ *  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ *  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+/** @jsxImportSource @emotion/react */
+
+import { css } from '@emotion/react';
+import Modal, { Styles } from 'react-modal';
+
+import { Theme } from '../theme';
+import MessageSquareWarning from '../theme/icons/MessageSquareWarning';
+import { useThemeContext } from '../theme/ThemeContext';
+
+Modal.setAppElement('body');
+
+const customStyles = (theme: Theme): Styles => ({
+	content: {
+		top: '50%',
+		left: '50%',
+		right: 'auto',
+		bottom: 'auto',
+		transform: 'translate(-50%, -50%)',
+		width: '40%',
+		maxHeight: '50%',
+		padding: 0,
+		overflow: 'hidden',
+		borderRadius: '10px',
+		border: '0.25px solid #000000',
+		boxShadow: '0px 4px 4px 0px #0000001A',
+		display: 'flex',
+		flexDirection: 'column',
+		backgroundColor: theme.colors.error_modal_bg,
+	},
+	overlay: {
+		backgroundColor: 'transparent',
+		zIndex: 1000,
+	},
+});
```

Comment:
please add a small comment that these are for the modal vs our normal Emotion css style. this has tripped me up on another project wondering why we're suddenly nesting objects and using strings